### PR TITLE
SAKIII-3918 Create other states of landing page carousel

### PR DESCRIPTION
**Pull request by bertpareyn**
https://jira.sakaiproject.org/browse/SAKIII-3918

### CODE REVIEW

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <% content_for :head do %>
     <%= stylesheet_link_tag "index" %>
+    <%= javascript_include_tag "jquery.jcarousel" %>
```
Comment (croby) [inline app/views/home/index.html.erb]: So I know you already did the work on this, but to get ahead of ourselves for the OAE work, can we instead use http://slidesjs.com/ for the carousel? I find it to be a much better, more mature library that is still being updated.

File: app/views/home/index.html.erb [hunk 2]
```diff
@@ -10,25 +12,46 @@
...
+                        <h1>Submitting your new widget couldn’t be simpler</h1>
+                        <h2>Once you’ve uploaded your widget, we will evaluate its suitability and code, and get back to you within a few days with a “yes’  or some suggested improvements.<a href="/submit" class="wl-link-button wl-button wl-overlay-button">Submit new widget</a></h2>
+                    </div>
+                </li>
+            </ul>
             <div class="carousel_choice">
-                <div class="carousel_choice_arrow"></div>
-                <a href="#" title="What is a Widget?">What is a Widget?</a>
+                <div class="carousel_choice_arrow" id="carousel_choice_arrow_1" style="display:none;"></div>
+                <a href="javascript:;" title="What is a Widget?" data-carousel-index="1" class="widgetlibrary_homecarousel_navitem">What is a Widget?</a>
```
Comment (croby) [inline app/views/home/index.html.erb]: Can we make these buttons?

Comment (bertpareyn) [inline app/views/home/index.html.erb]: The new Slider plugin requires links and href tags for pagination so I left them as they are.

File: app/views/home/index.html.erb (hunk 1 above)
Comment (bertpareyn) [inline app/views/home/index.html.erb]: Switched around and you're right. This plugin is way easier to use and the end result looks polished.

**Comment by bertpareyn** [comment]
@croby: Updated pull request

**Pull request merged**